test/manual/amp-ad.yieldbot.amp.html

```diff
@@ -27,7 +27,7 @@
           data-psn="1234"
           data-yb-slot="medrec"
           data-slot="/2476204/medium-rectangle"
-          data-multi-size="300x220,300x200"
+          data-multi-size="300x220,300x210, 300x200,300NNN,300xNNN,"
```

Comment:
The input is invalid. Is it for testing only? 
Also we do not require test page for individual ad. Can the example in ads.amp.html replace this file?